spec/30-processing-model.md

```diff
@@ -0,0 +1,52 @@
+# Processing Model
+
+This section provides a step-by-step description of the behavior of a tracing implementation - also called tracer -  when a request is received. This description can be used as a reference when implementing a tracecontext compliant tracing system, middleware or cloud service.
+
+## Processing Model for Working with Trace Context
+
+The processing model describes the behaviour of a tracer which forwards **and** modifies
+tracecontext headers.
```

Comment:
should we use fields instead of headers? So this section will be more like a semantical model that includes other protocols?

Reply:
the main document is still about HTTP so I think headers is fine b/c that's what it is. Alternative protocols should define how they translate headers to other transport mechanisms

Reply:
it's fine to keep headers. Maybe later down the release cycle we can address this. Now we have them as fields in spec in some places and headers in others